Assignments/REACT/Pratik Ivnathe/React Js Assignment/Pratik_Q.3.js

```diff
@@ -0,0 +1,11 @@
+function Greeting(props) {
+    return (
+      <div>
+        <h2>Good Afternoon,{props.name}</h2>
+        <p>this is paragraph</p>
+      </div>
+    ); 
+  }
+  const comp_fun_element=<Greeting name = "Ajay "/>
+  ReactDOM.render(<Greeting />, document.getElementById("root"));
```

Comment:
in render method pass comp_fun_element.ReactDOM.render(comp_fun_element, document.getElementById("root"));